src/components/vr-scatter-plot/vr-scatter-plot.component.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <a-scene></a-scene>
+    </body>
+</html>
```

Comment:
Now that it is part of Angular component, I think it can simply have
```ts
<a-scene></a-scene>
```

without `<html>` and `<body>` tags.